lib/datastore/request.js

```diff
@@ -400,7 +401,7 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  * //-
  * var query = datastore.createQuery('Lion');
  *
- * datastore.runQuery(query, function(err, entities) {});
+ * datastore.runQuery(query, function(err, entities, info, apiResponse) {});
```

Comment:
Since we force the autobatching, I would probably vote for none at all until we get a FR with a case asking for it (then we'll know what people are looking for).